plugins/item_tasks/web_client/stylesheets/paginateTasksWidget.styl

```diff
@@ -4,3 +4,12 @@
   .g-task-pagination
     ul
       margin 0
+
+.g-list-group-item
+  color #555
+  position relative
+  display block
+  padding 10px 15px
+  margin-bottom -1px
```

Comment:
Is there a way to cleanly reduce the `margin-top` of the next element instead? Negative margins scare me.